pkg/sip/client.go

```diff
@@ -20,9 +20,9 @@ import (
 	"log/slog"
 	"sync"
 
-	"github.com/emiago/sipgo"
-	"github.com/emiago/sipgo/sip"
 	"github.com/frostbyte73/core"
+	"github.com/livekit/sipgo"
+	"github.com/livekit/sipgo/sip"
```

Comment:
Cosmetics, but we usually put livekit module imports in a different section than 3rd party.